README.md

```diff
@@ -152,11 +152,10 @@ Simply add skipBabel to your global variables under the `ts-jest` key:
 }
 ```
 
-### Use customized .babalrc
-By default, `ts-jest` use only the embedded .babelrc file.
-It should work in most cases, and keep configuration simple.
-In case your tests need special .babelrc settings to work,
-set `"useBabelrc": true` to tell `ts-jest` read your .babelrc.
+### Using `.babelrc`
+
+When using Babel, ts-jest, by default, doesn't use the `.babelrc` file. If you want ts-jest to use `.babelrc`, you should set the `globals > ts-jest > useBabelrc` flag to `true`.
```

Comment:
When referring to the package, I'd prefer ts-jest without any formatting.

When referring to the section key (in jest config) I'd prefer it to be `ts-jest`